transaction/account/type_update_contract.go

```diff
@@ -0,0 +1,85 @@
+package account
+
+import (
+	"fmt"
+
+	"github.com/Concordium/concordium-go-sdk"
+)
+
+type UpdateContractResultEvent struct {
+	*concordium.TransactionResultEvent `json:""`
+
+	ContractVersion int                         `json:"contractVersion"` // Updated
+	Address         *concordium.ContractAddress `json:"address"`         // Updated, Interrupted, Resumed
+	Instigator      *concordium.Address         `json:"instigator"`      // Updated
+	Amount          *concordium.Amount          `json:"amount"`          // Updated, Transferred
+	Message         concordium.Model            `json:"message"`         // Updated
+	ReceiveName     concordium.ReceiveName      `json:"receiveName"`     // Updated
+	Events          []concordium.Model          `json:"events"`          // Updated, Interrupted
```

Comment:
Events should be of a different type than `Model`. They are a byte array, but the meaning is different.